dms-infrastructure/src/main/kotlin/team/aliens/dms/persistence/remain/RemainOptionPersistenceAdapter.kt

```diff
@@ -23,4 +23,9 @@ class RemainOptionPersistenceAdapter(
     override fun queryRemainOptionById(remainOptionId: UUID) = remainOptionMapper.toDomain(
         remainOptionRepository.findByIdOrNull(remainOptionId)
     )
+
+    override fun queryAllRemainOptionBySchoolId(schoolId: UUID) = remainOptionRepository.findAllBySchoolId(schoolId)
+        .map {
+            remainOptionMapper.toDomain(it)!!
+        }
```

Comment:
```suggestion
    override fun queryAllRemainOptionBySchoolId(schoolId: UUID) =
        remainOptionRepository.findAllBySchoolId(schoolId)
            .map { remainOptionMapper.toDomain(it)!! }
```
p3 이거 줄바꿈 좀 애매한데 이렇게 바꾸는거 어떠신가요?